.agents/scripts/voice-bridge.py

```
@@ -0,0 +1,834 @@
+            req = urllib.request.Request(self.server_url, method="HEAD")
+            urllib.request.urlopen(req, timeout=2)
+            self.use_attach = True
+            log.info(f"OpenCode server found at {self.server_url}")
+        except Exception:
+            self.use_attach = False
+            log.info("No OpenCode server found, will use standalone mode")
+
+    def _start_server(self):
+        """Start opencode serve in background."""
+        log.info(f"Starting opencode serve on port {self.server_port}...")
+        proc = subprocess.Popen(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)